yarn.lock

```diff
@@ -920,7 +920,7 @@
   dependencies:
     eslint-visitor-keys "^3.3.0"
 
-"@eslint-community/regexpp@^4.4.0", "@eslint-community/regexpp@^4.6.1":
+"@eslint-community/regexpp@^4.5.1", "@eslint-community/regexpp@^4.6.1":
```

Comment:
I used "yarn add" to update each of our modules and so this yarn.lock file was automatically updated. I spot checked it and it seems like its all good even though a number of transitive dependencies got updated.